### DISCUSSION TITLE
TF3 Future

### DISCUSSION THREAD

**Discussion by foubou** [Kaplas80/TF3#19] [General]
Hi Kaplas ! Thank you for your hard work and the tools you provide to the community.
Are you still working on TF3 ? There were no updates lately so I was wondering if you were still working on it.
Also, do you intend to add support for AI The Somnium Files Nirvana Initiative ?
You successfully decompiled AITSF1 in TF2.
There are high chances AITSF1 and AITSF2 use the same engine and same file system, so maybe it will be easier for you to work on AITSF2, since you already worked on AITSF1.
Thank you for your answer.

**Comment by Kaplas80** [comment]
Hello foubou!

It is not abandoned but I don't have so much free time as before, so the progress is a lot slower now.

About supporting other games, right now I'm working to add "The Legend of Heroes: Trails in the Sky SC", but I don't think it will be ready soon. Once I finish with it, maybe I can look into AITSF2.

**Comment by foubou** [comment]
Good news ! I am very glad to learn you are working on The Legend of Heroes: Trails in the Sky SC !
Thank you very much for considering working on AITSF2 too !

**Comment by foubou** [comment]
Hi Kaplas. Someone worked on decompiling the scenario files of AITSF2-Nirvana Initiative.
It's a simple drag and drop tool, converting the game text files into JSON format.
https://github.com/SutandoTsukai181/ai2_msgpack/releases/
Feel free to take a look at it (if you are still willing to work on AITSF2 like you did for AITSF1 of course).
Thank you !
